src/main/java/jeju/controller/MypageController.java

```diff
@@ -1,16 +1,34 @@
 package jeju.controller;
 
+import javax.servlet.http.HttpServletRequest;
+
 import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
 
 @Controller
 public class MypageController {
 
 	@GetMapping("/mypage/user")
-	public String userpage() {
+	public String userpage(HttpServletRequest request, Model model) {
+		
+    String currentUrl = request.getRequestURI();
+
+    model.addAttribute("currentUrl", currentUrl);
 		return "mypage/userpage";
 	}
```

Comment:
들여쓰기가 안 된 것 같은데, 이왕이면 잘 맞춰서 통일해주면 보기 더 좋을것 같아요!
```suggestion
		
                String currentUrl = request.getRequestURI();

                model.addAttribute("currentUrl", currentUrl);
		return "mypage/userpage";
	}
```